src/sentry/web/urls.py

```diff
@@ -366,7 +369,7 @@ def init_all_applications():
     # Keep named URL for for things using reverse
     url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_id>[\w_-]+)/issues/(?P<group_id>\d+)/$', react_page_view,
         name='sentry-group'),
-    url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_id>[\w_-]+)/$', react_page_view,
+    url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_slug>[\w_-]+)/$', ReactProjectPageView.as_view(),
```

Comment:
I'm still going to push back and say we keep `project_id` for now. We can plan to change everything to `project_slug` later, but this is pretty far outside the scope of this PR, which is already large enough. Changing to `project_slug` is only going to add a bunch more noise to this.